gluonnlp/model/train/language_model.py

```diff
@@ -25,6 +25,157 @@
 from gluonnlp.model.utils import _get_rnn_layer
 from gluonnlp.model.utils import apply_weight_drop
 
+class MOSRNN(Block):
```

Comment:
Please add test cases to test the forward of the network, especially on the last mixture softmax layer.